### DISCUSSION TITLE
New Source: Sprout Social

### DISCUSSION THREAD

**Discussion by rludington** [airbytehq/airbyte#35217] [New Connector Request]
## Tell us about the new connector you’d like to have
We would like [Sprout Social](https://sproutsocial.com/) added as a new source connection for Airbyte. My team specifically wants to move that data to a GCP warehouse. We do not need support for an older version of the API, we are content with the current API version. 

## Describe the context around this new connector
Sprout Social allows us to consolidate all of our social media in one place which also provides the opportunity to query all of our social media information in one place. Our aim is to understand how our social media marketing is impacting sales. While more syncs are typically better, a 24 hour sync is sufficient to begin answering questions. We would like to get information from all current platforms offered on the API: 

- Post Performance Report
- Facebook Pages Report
- Instagram Business Profiles Report
- Twitter Profiles Report
- LinkedIn Pages Report

We are willing to develop this connector, as the potential benefit is great. This is an instance where ad hoc reporting is too cumbersome given the number of social media campaigns, though ad hoc querying is possible in any number of ways using their API.

**Comment by rludington** [comment]
I would like to work on this, but am unable to assign it to myself

**Comment by YowanR** [comment]
@rludington are you still interested in working on this one?

**Comment by misteryeo** [comment]
**Issue was linked to Harvestr Discovery:**  [New Source: Sprout Social](https://app.harvestr.io/components/0/list/iDAoorpkV)

[a comment by an automated account is not shown]
